tensilelite/Tensile/SolutionStructs.py

```diff
@@ -3527,12 +3527,9 @@ def subCheckLdsBlockSizePerPad(tc, idx):
       # TODO: support ONLL if necessary
       state["OptNoLoadLoop"] = 0
 
-    if not state["ProblemType"]["GroupedGemm"]:
-      if state["ProblemType"]["SupportUserArgs"]:
-        reject(state, "Currently only grouped gemm supports SupportUserArgs.")
-    if state["GlobalSplitU"] > 1:
-      if state["ProblemType"]["SupportUserArgs"] and state["_GlobalAccumulation"] != 'MultipleBufferSingleKernel':
-        reject(state, "Currently SupportUserArgs does not support GSU > 1.")
+    # if state["GlobalSplitU"] > 1:
```

Comment:
Why not remove this code?

Reply:
if we don't have this restriction now, why did you want to keep it to remind you?